lib/smart_app_launch/token_introspection_group_stu2_2.rb

```diff
@@ -39,6 +39,11 @@ class SMARTTokenIntrospectionGroupSTU22 < SMARTTokenIntrospectionGroup
 
       See the individual test groups for more details and guidance.
     )
+
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@270',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@273',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@282'
```

Comment:
@270 - remove - client requirement in server test
@273 - move into the "Token introspection response for an active token contains required fields" test of token_introspection_response_group.rb
@282 - not sure, probably remove, but if you see a place where this is tested, could move it to that specific test